react/components/EmptyState/index.js

```diff
@@ -6,11 +6,9 @@ class EmptyState extends PureComponent {
     const { title, children, testId } = this.props
 
     return (
-      <div
-        className="MAIN br3 flex c-muted-2 justify-center pv9 ph6 ph9-l tc"
```

Comment:
@kevinch any reason for this class `MAIN` added [here](https://github.com/vtex/styleguide/commit/8f92d899453238bc28867c47b4d114c9ed5bc3d5)?

Reply:
Can't remember. I'd say it was a debug class helper when developing the table.